Backend/models.py

```diff
@@ -324,6 +324,7 @@ class UserTransactions(db.Model):
     user = db.relationship('User', backref='users_transactions', lazy=True)
 
 
+
     def __init__(self, id, type, description, amount, status, date, time):
         self.id = id
```

Comment:
remove self.id from all modal classes __init__ functions

Reply:
a lot of modifications had been made to the model.py at my own end though
I now linked the wallet and Details table to User and company and then linked Wallet and Transaction table

Reply:
Okay let's keep in sync, create a pr with the updated models, we'll merge it then I'll pull to my branch, then update my pr

Reply:
ok